helpdesk_mgmt/security/helpdesk_security.xml

```diff
@@ -32,6 +32,12 @@
         <field name="domain_force">['|',('user_id','=',user.id),('user_id','=',False)]</field>
         <field name="groups" eval="[(4, ref('group_helpdesk_user_own'))]"/>
     </record>
+    <record id="helpdesk_all_tickets_rule" model="ir.rule">
```

Comment:
You are introducing here other features not related to multi-company. Please remove this and the other one for being discussed in other PR.